src/pages/[course_name]/prompt.tsx

```diff
@@ -1343,9 +1438,14 @@ export const showToastNotification = (
   isError = false,
   icon?: React.ReactNode,
 ) => {
+  // Calculate duration based on message length (minimum 5 seconds, add 1 second for every 20 characters)
```

Comment:
that's a pretty nifty way to determine how long to keep a notification open. wow cool